tests/jballerina-unit-test/src/test/resources/test-src/enums/enums.bal

```diff
@@ -115,13 +115,21 @@ function testEnumInRecursiveArray() {
     Keyword u = "union";
     assert(u, "union");
 
-    ApproxType b = "'boolean";
-    assert(b, 'boolean);
+    ApproxType b = "boolean";
+    assert(b, "boolean");
```

Comment:
Shall we keep the previous version also? 

```ballerina
    assert(b, 'boolean);
```

If there's a bug in the assignability check for `ApproxType b = "boolean";`, we may not be asserting the actual value of `'boolean`.